Description: Add Keywords entry into desktop file
 Adding Keywords entry to facilitate searching through entries

Forwarded: not-needed
Author: Elías Alejandro Año Mendoza <ealmdz@gmail.com>
Last-Update: 2015-08-09

--- uget.orig/uget-gtk.desktop
+++ uget/uget-gtk.desktop
@@ -7,4 +7,4 @@ Icon=uget-icon
 Terminal=false
 Type=Application
 Categories=Network;FileTransfer;
-
+Keywords=filetransfer;download files;download manager;
